OpenRA.Game/Settings.cs

```diff
@@ -90,18 +90,36 @@ public class DebugSettings
 		public bool SanityCheckUnsyncedCode = false;
 		public int Samples = 25;
 		public bool IgnoreVersionMismatch = false;
+
+		[Desc("Dump performance data into cpu.csv and render.csv accessible via ~/.openra/Logs.")]
```

Comment:
This will be read in the Linux man page only. I will add a console text which will tell the real location when activated.